<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPersonDialog</class>
 <widget class="QDialog" name="AddPersonDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Προσθήκη Ατόμου</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>res/app_icons/plus.png</normaloff>res/app_icons/plus.png</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>441</width>
     <height>331</height>
    </rect>
   </property>
   <property name="title">
    <string>Βασικά Στοχεία</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>421</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Όνομα:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="surnameLabel">
       <property name="text">
        <string>Επώνυμο:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="surnameEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="idLabel">
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
       <property name="text">
        <string>Μοναδικό Αναγνωριστικό (ID):</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="idEdit">
       <property name="inputMethodHints">
        <set>Qt::ImhDigitsOnly</set>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="birthLabel">
       <property name="text">
        <string>Ημερομηνία Γέννησης:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QDateEdit" name="dateEdit"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="lnameLabel">
       <property name="text">
        <string>Όνομα (Λατινικά):</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="lnameEdit"/>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="lsurnameLabel">
       <property name="text">
        <string>Επώνυμο (Λατινικά):</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLineEdit" name="lsurnameEdit"/>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLabel" name="surnameErrorLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="nameErrorLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QLabel" name="idErrorLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="11" column="0" colspan="2">
      <widget class="QLabel" name="lsurnameErrorLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="2">
      <widget class="QLabel" name="lnameErrorLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QLabel" name="dateErrorLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>441</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Προσθήκη Φωτογραφίας</string>
   </property>
   <widget class="QToolButton" name="addProfileButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>res/app_icons/folder.png</normaloff>res/app_icons/folder.png</iconset>
    </property>
   </widget>
   <widget class="QLineEdit" name="profilePathDisplay">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>381</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="imgErrorLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>381</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>460</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="saveButton">
      <property name="text">
       <string>Αποθήκευση</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Ακύρωση</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
